1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
+    "\n",
+    "HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities, and it will assist independent living at home for people with disabilities.\n",
+    "\n",
+    "In this notebook, you will learn the overview of the robot by using the HSR simulator."
```

Comment:
"In this notebook, you will learn about the robot by using the HSR simulator."

I removed the expression "learn the overview of" because it felt a bit unnatural.